content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/serial_numbers.rst

```diff
@@ -1,147 +1,320 @@
+   validated without a serial number being assigned.
 
-For each type, you can decide if you allow the creation of new lot
-numbers or want to use existing ones. By default, the creation of new
-lots is only allowed at product reception.
+.. image:: serial_numbers/serial-numbers-user-error-popup.png
+   :align: center
+   :alt: User error popup prompting serial number entry.
 
-.. image:: serial_numbers/serial_numbers_11.png
-    :align: center
+From here, click the :guilabel:`Additional Options` menu, represented by a `hamburger` icon (four
+vertical lines, located to the right of the :guilabel:`Unit of Measure` column in the
+:guilabel:`Operations` tab). Clicking that icon reveals a :guilabel:`Detailed Operations` pop-up.
 
-.. tip::
-      If you have inter-warehouse transfers and track products by lots, it can
-      be useful to allow using existing lot numbers in receipts too.
+In this pop-up, configure a number of different fields, including the assignation of a serial
+number (or serial numbers) under the :guilabel:`Lot/Serial Number Name` column, located at the
+bottom of the pop-up.
+
+There are three ways to do this: manually assigning serial numbers, automatically assigning serial
+numbers, and copy/pasting serial numbers from an Excel file.
+
+Assign serial numbers manually
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To assign serial numbers manually, click :guilabel:`Add a line` from the :guilabel:`Detailed
+Operations` pop-up, and first choose the location where the product will be stored under the
+:guilabel:`To` column.
+
+Then, type a new :guilabel:`Serial Number Name`, and set the :guilabel:`Done` quantity in the
+appropriate columns.
+
+Repeat this process for the quantity of products shown in the :guilabel:`Demand` field, and until
+the :guilabel:`Quantity Done` field displays the correct (matching) number of products processed.
+
+Assign serial numbers automatically
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If a large quantity of products need individual serial numbers assigned to them, Odoo can
+automatically generate and assign serial numbers to each of the individual products.
+
+To accomplish this, start with the :guilabel:`First SN` field in the :guilabel:`Detailed
+Operations` pop-up window, and type the first serial number in the desired order to be assigned.
 
-Serial Number traceability
+Then, in the :guilabel:`Number of SN` field, type the total number of items that need newly
+generated unique serial numbers assigned to them.
+
+Finally, click :guilabel:`Assign Serial Numbers`, and a list will populate with new serial numbers
+matching the ordered quantity of products.
+
+.. image:: serial_numbers/serial-numbers-auto-assign-sn.png
+   :align: center
+   :alt: Automatic serial number assignment in detailed operations popup.
+
+Copy/paste serial numbers from an Excel file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To copy and paste serial numbers from an existing *Excel* file, first populate an *Excel*
```

Comment:
I don't think we need to italicize Excel, curious on @StraubCreative 's thoughts here 🤔